### DISCUSSION TITLE
[Meta] Best way for making feature requests?

### DISCUSSION THREAD

**Discussion by HarrisonLeece** [FreeCAD/FreeCAD#12963] [Code contribution discussions]
I am more concerned with using FreeCAD than making my own pull requests, but if myself or another user has a simple feature request (in this case, FreeCAD-->PardDesign--Measure Linear--> double clicking circular edge indicates diameter and radius) how can I make the request without disrupting other actions?

This could be a solution both on github and off of github, such as freeCAD forums.

**Comment by pierreporte** [comment]
You need to create an issue on Github. It is best to word it as a problem report.

@maxwxyz

**Reply by HarrisonLeece** [reply]
@pierreporte I think this will be an amazing addition to freeCAD, I anticipate its release.  Closing
